beacon_node/eth2_libp2p/src/peer_manager/peerdb.rs

```diff
@@ -181,20 +210,27 @@ impl<TSpec: EthSpec> PeerDB<TSpec> {
     /// temporarily.
     ///
     /// This is used to determine if we should accept incoming connections or not.
-    pub fn is_banned(&self, peer_id: &PeerId) -> bool {
```

Comment:
Not sure how much of a pain this would be, but maybe this function should be called something like `ban_status` now that it no longer returns a bool. Or at least the doc comment should be updated